reconcile/utils/slack_api.py

```diff
@@ -342,3 +355,48 @@ def _get(self, resource: str) -> Dict[str, Any]:
 
         self._results[resource] = results
         return results
+
+    @classmethod
+    def create_using_queries(cls, integration_name: str,
```

Comment:
Wouldn't `create_from_queries` be more consistent with `create_from_dict`?

Reply:
--> moved to slack_base.py and renamed